WhoIsThat/WhoIsThat/HomePage.xaml.cs

```diff
@@ -9,12 +12,52 @@
 
 namespace WhoIsThat
 {
+
 	[XamlCompilation(XamlCompilationOptions.Compile)]
 	public partial class HomePage : ContentPage
 	{
 		public HomePage ()
 		{
 			InitializeComponent ();
 		}
+
+        private async void takePicture(object sender, EventArgs e)
+        {
+            //Requesting permissions to user camera and storage, too lazy to implement checks if user already have those, later 'todo'
```

Comment:
This could probably be moved into separate manager which deals with images, not sure whether this `CrossMedia` is not  a part of Activity

Reply:
But this is optional for now 😄

Reply:
I was planning on creating some sort of ImageHandler and decided to do that later with backend functionality. Not sure about return type, should it be MediaFile or some stream. But this is optional for now, right :D